larray/util/misc.py

```diff
@@ -81,18 +81,23 @@ def prod(values):
     return reduce(operator.mul, values, 1)
 
 
-def format_value(value, missing, fullinfo=False):
+def format_value(value, missing, fullinfo=False, precision=None):
     if isinstance(value, float) and not fullinfo:
         # nans print as "-1.#J", let's use something nicer
         if value != value:
             return missing
+        elif precision is not None:
+            return '{:2.{precision}f}'.format(value, precision=precision).rstrip('0').rstrip('.')
```

Comment:
concerning ``table2str`` and ``format_value``, why did you choose to not rely on ``np.array2string`` for printing the internal data? 
Because we could benefit of all printing options provided by ``np.set_printoptions``.

Reply:
yes, but IIRC (I wrote this a **long** time ago), it was simpler not to use it because combining with the headers wasn't that easy I think. Especially computing the maximum width of each column (including its header !!) of the 2D "table" was problematic when using np.array2string. IIRC, I also could not find an option to remove the `[` `]` from the string that numpy produces. Now if we support more of the options of set_printoptions that numpy has, it **might** be worth it but currently I don't think it is.